src/shared/constants/config/routerPaths.ts

```diff
@@ -221,6 +226,10 @@ const pathsMap = {
 
   userProfile: (userID: string) => `/${PathName.userProfile}/${userID}`,
   userProfileSettings: (userID: string) => `/${PathName.userProfile}/${userID}/${PathName.userProfileSettings}`,
+  userProfileSettingsGeneral: (userID: string) =>
+    `/${PathName.userProfile}/${userID}/${PathName.userProfileSettingsGeneral}`,
+  userProfileSettingsNotifications: (userID: string) =>
+    `/${PathName.userProfile}/${userID}/${PathName.userProfileSettingsNotifications}`,
```

Comment:
Based on the last comment these become,

```suggestion
  userProfileSettingsGeneral: (userID: string) =>
    `/${PathName.userProfile}/${userID}/${PathName.userProfileSettings}/${PathName.userProfileSettingsGeneral}`,
  userProfileSettingsNotifications: (userID: string) =>
    `/${PathName.userProfile}/${userID}/${PathName.userProfileSettings}/${PathName.userProfileSettingsNotifications}`,
```